lightmdb/models/user.py

```diff
@@ -70,6 +70,18 @@ def followers(self):
             return result[0]
         return 0
 
+    def get_watched_movies(self):
+        db = get_database()
+        cursor = db.cursor
+        cursor.execute("SELECT * FROM {table} WHERE id in (SELECT movie_pk from watched_movies where id = %(pk)s)".format(
+            table="movies"),
+            {'pk': self.pk}
+        )
+        result = cursor.fetchone()
+        if result:
+            return result
```

Comment:
veri tabani yerine gelen cevabi for dongusu ile obje olusturup array-e ekle, toplu halde return et. User ve Movie-de ornegi olmasi lazim. filter de boyle calisiyor.